### DISCUSSION TITLE
URL parameter for "new" projects when creating an issue doesn't work

### DISCUSSION THREAD

**Discussion by thromera** [community/community#41738] [Projects and Issues] [answered]
### Select Topic Area

Bug

### Body

On projects (classic) it is possible to pre-fill the project using a parameter URL. 

With classic projects, this issue URL: 
https://github.com/my-org/my-repo/issues/new?assignees=&labels=bug&template=bug_report.md&projects=my-org/50

correctly pre-fills the `project` field on the issue: 

<img width="1335" alt="image" src="https://user-images.githubusercontent.com/1775878/207633338-e9f8447a-0286-49b5-bf8c-58ff7299a758.png">

It doesn't work with non-classical projects: 
https://github.com/my-org/my-repo/issues/new?assignees=&labels=bug&template=bug_report.md&projects=my-org/61

![image](https://user-images.githubusercontent.com/1775878/207901732-cecd40ef-994e-45f4-b476-93933ba602ad.png)

Happy to disclose privately the repositories / org to a GH employee if necessary.

**Comment by rileybroughten** [comment, marked as answer]
Hello! 👋 This is not currently supported for Projects in comparison to Projects (classic), but there is another issue for the same request linked here https://github.com/community/community/discussions/19811. 

I will close out this issue as a duplicate, and you can follow along on the above issue as updates will be provided in a single place! Thank you! ✨
